### PR TITLE
fix: Bump payments-sdk to address gas limit issue

### PR DESCRIPTION
**Pull request by dckesler**
### Description

The payments-sdk needed to be updated to prevent a gas limit issue on mainnet

### Other changes

We also commented out the country info for now

### Tested

Was able to send merchant payments with the new sdk.

### How others should test

Running on mainnet go to https://prod-celo-demo-store.demo.firstdag.com/, pick a product, scan the QR code and verify it sends.

### Backwards compatibility

Yes

### CODE REVIEW

File: packages/mobile/src/merchantPayment/MerchantPaymentScreen.tsx [hunk 1]
```diff
@@ -137,8 +137,8 @@ function MerchantPaymentScreen({ route }: Props) {
                   {businessInformation.address.city}, {businessInformation.address.state + ' '}
                   {businessInformation.address.postalCode}
                   {'\n'}
-                  {businessInformation.address.country}
-                  {'\n'}
+                  {/*businessInformation.address.country*/}
+                  {/*'\n'*/}
```
Comment (jeanregisser) [inline packages/mobile/src/merchantPayment/MerchantPaymentScreen.tsx:141]: Just checking, is it intentional?

**Review by jeanregisser — APPROVED** [review]
LGTM 👍

**Review by jeanregisser — APPROVED** [review]

**Comment by MuckT** [comment]
Was able to send payments when testing locally on mainnet.

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified above task on Latest Test Flight Release build V1.29.0, Android Internal Release Build V1.29.0 & updated status in test rails.

- On scanning QR code from merchant payment [website](https://sandbox-celo-demo-store.demo.firstdag.com/), from account having valid token balance, the transaction get successfully completed and user is redirected to home screen and activity appears in the app, But on the web page "Processing payment" popup appears.
- When user scan QR code without phone number verification it asks to connect phone number for purchasing a product, hence we have raised a bug for that.

**Reference Link**: [Link](https://github.com/valora-inc/wallet/issues/2223)
**New Bug** : [link](https://github.com/valora-inc/wallet/issues/2224)

Test rail link:
**Android 11:** https://valoraapp.testrail.io/index.php?/tests/view/20951
**iOS13:** https://valoraapp.testrail.io/index.php?/tests/view/20888
**iOS14:** https://valoraapp.testrail.io/index.php?/tests/view/20825

**Devices:** Google Pixel 2XL (11.0), iPhone SE 2020 (14.8.1), iPhone 7 (13.3)
**cc:** @kathaypacific
Thanks..!!

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified above task on Latest Android Internal Release Build V1.29.2 & updated status in test rails.

**Observations:**
- Loader is appearing in Estimated fees section for fees on Sending screen when user scans QR code from merchant payment website. "Your payment failed to submit" error appears , transaction fails & user redirects to home screen when user taps on Send button and complete sending flow after scanning QR code from merchant payment website, below is the link :
**Reference Link:**[Link](https://github.com/valora-inc/wallet/issues/2155#issuecomment-1083050068) 

-“There was an error loading..” error is displayed when the user tries to scan the QR code from Merchant Payment url while creating the account while skipping the phone number. We have filed a bug for the same.

**New Bug** : [link](https://github.com/valora-inc/wallet/issues/2275)

**Test rail link:**
**Android 12:** https://valoraapp.testrail.io/index.php?/tests/view/21014

**Devices:** Google Pixel 4a (12.0)
**cc:** @dckesler 
Thanks..!!

**Pull request merged**